### PR TITLE
Activated "Pending Reviews" menu item when merchant opens 'Pending Reviews' section

### PR DESCRIPTION
**Pull request by rav-redchamps**
### Description (*)

This pull request will activate the "Pending Reviews" menu item when merchant opens admin path 'Marketing > User Content > Pending Reviews'
### Fixed Issues (if relevant)

1. magento/magento2#26229 >: Active menu is not set when opening admin path Marketing > User Content > Pending Reviews

### Manual testing scenarios (*)

1. Go to admin path 'Marketing > User Content > Pending Reviews' & notice that 'Marketing' menu item is now shown as active

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by rav-redchamps** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rav-redchamps. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rav-redchamps, here is your [new Magento instance](https://pr-26230.instances.magento-community.engineering).
Admin access: [https://pr-26230.instances.magento-community.engineering/admin](https://pr-26230.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rav-redchamps** [comment]
Verified on the test instance. Change works well

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
Hello @rav-redchamps, thank you for your contribution. Due to [Magento Definition of Done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html#automated-tests) the changes should be covered by tests. Could you please cover your fix by automated tests?
I think the functional test is preferable in this particular case.
Thank you!

**Comment by rav-redchamps** [comment]
> Hello @rav-redchamps, thank you for your contribution. Due to [Magento Definition of Done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html#automated-tests) the changes should be covered by tests. Could you please cover your fix by automated tests?
> I think the functional test is preferable in this particular case.
> Thank you!

I just added the function test. Please feel free to let me know if anything else is required.

**Comment by engcom-Echo** [comment]
@magento run all tests

File: app/code/Magento/Review/Test/Mftf/Test/AdminMarketingPendingReviewsNavigateMenuTest.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminMarketingPendingReviewsNavigateMenuTest">
+        <annotations>
+            <features value="Pending Reviews"/>
+            <stories value="Menu Navigation"/>
+            <title value="Admin marketing pending reviews navigate menu test"/>
+            <description value="Admin should be able to navigate to Marketing > Pending Reviews"/>
+            <severity value="CRITICAL"/>
+            <testCaseId value="MC-14200"/>
```
Comment (dmytro-ch) [inline app/code/Magento/Review/Test/Mftf/Test/AdminMarketingPendingReviewsNavigateMenuTest.xml]: Is this internal ticket id relevant to this test?

**Review by dmytro-ch — COMMENTED** [review]
Hi @rav-redchamps. Sorry for the late reply and thank you for your patience.
Could you please add at least one assertion to your test which covers the issue fixed in this PR. Currently it does not actually verify whether the menu item is active or not.
Thank you!

**Comment by engcom-Echo** [comment]
I will take to fix MFTF

**Comment by dmytro-ch** [comment]
@engcom-Echo, great, thank you! :)

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6938](https://jira.corp.magento.com/browse/ENGCOM-6938) has been created to process this Pull Request

**Review by dmytro-ch — DISMISSED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6938](https://jira.corp.magento.com/browse/ENGCOM-6938) has been created to process this Pull Request

**Review by dmytro-ch — DISMISSED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6938](https://jira.corp.magento.com/browse/ENGCOM-6938) has been created to process this Pull Request

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6938](https://jira.corp.magento.com/browse/ENGCOM-6938) has been created to process this Pull Request

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6938](https://jira.corp.magento.com/browse/ENGCOM-6938) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6938](https://jira.corp.magento.com/browse/ENGCOM-6938) has been created to process this Pull Request

**Pull request merged**